docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -30,6 +30,50 @@ psql -h database.example.com -U username ... -f kubearchive.sql
 This file creates a PostgreSQL schema, a table for KubeArchive and other stuff KubeArchive
 requires.
 
+== Configure Knative Eventing
+
+By default, Knative Eventing use `MTChannelBasedBroker` as the class for any Broker that does not explicitly define its
+class and uses `InMemoryChannel` as the backing channel implementation. Knative Eventing strongly recommends using a
+native broker implementation for production workloads instead. We recommend the Apache Kafka Broker.
+
+. If you need a Kafka cluster, follow the
+link:https://strimzi.io/docs/operators/latest/deploying#con-strimzi-installation-methods_str[Strimzi deployment instructions]
+. Follow the
+link:https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/#installation[Knative Eventing Kafka Broker installation instructions]
+. Create a ConfigMap named kafka-broker-config in the knative-eventing namespace. Use
+link:https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/#configure-a-kafka-broker[this ConfigMap]
+as a guide.
+. Update the config-br-defaults ConfigMap in the knative-eventing namespace
++
+[source,bash]
+----
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  default-br-config: |
+    clusterDefault:
+      brokerClass: MTChannelBasedBroker
+      apiVersion: v1
+      kind: ConfigMap
+      name: config-br-default-channel
+      namespace: knative-eventing
+    namespaceDefaults:
+      kubearchive:
+        brokerClass: Kafka
+        apiVersion: v1
+        kind: ConfigMap
+        name: kafka-broker-config
+        namespace: knative-eventing
+EOF
+----
+
+Knative Eventing supports other Broker types as well. You can learn more about them
+link:https://knative.dev/docs/eventing/brokers/broker-types/[here]
```

Comment:
I think this is too much information. If you are willing to reduce to information then I would go with:

```suggestion
KubeArchive uses a memory based  broker to avoid missing Kubernetes resource updates. If this broker is not performant enough for your use case, check
link:https://knative.dev/docs/eventing/brokers/broker-types/[Knative's Eventing supported broker types].
We recommend you use the Kafka broker.
```

If the change is more complicated that this I think it needs its own page then.

Reply:
Let's not recommend anything.  We can say we use a Kafka broker, I suppose.

I agree with less information here, but I think we can say we use the defaul `MTChannelBasedBroker` but that Knative recommends one of the other brokers in production.

Reply:
I shortened this section mostly following @rh-hemartin suggestion but moved in the bit about other broker types into an admonition and followed @ggallen suggestion about not recommending anything and instead worded it as knative eventing recommends